contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -356,6 +360,20 @@ contract KeeperRegistryDev is
     LINK.transfer(to, amount);
   }
 
+  /**
+   * @notice allows the admin of an upkeep to modify gas limit
+   * @param id upkeep to be change the gas limit for
+   * @param gasLimit new gas limit for the upkeep
+   */
+  function setGasLimit(uint256 id, uint32 gasLimit) external override onlyActiveUpkeep(id) onlyUpkeepAdmin(id) {
```

Comment:
can we nix this function and just set the limit in `setConfig`?

Reply:
This is setting limit for upkeep, will make the name clearer